docs/eventing/getting-started.md

```diff
@@ -0,0 +1,496 @@
+---
+title: "Getting Started with Eventing"
+linkTitle: "Getting Started"
+weight: 9
+type: "docs"
+---
+
+Use this guide to learn how to create, send, and verify events in Knative. The steps in this guide demonstrate a basic developer flow for managing events in Knative, including:
+
+- [Installing the Knative Eventing component](#installing-knative-eventing)
+- [Creating and configuring Knative Eventing Resources](#setting-up-knative-eventing-resources)
+- [Sending events with HTTP requests](#sending-events-to-the-broker)
+- [Verifying events were sent correctly](#verifying-events-were-received)
+
+## Before you begin
+
+To complete this guide, you will need the following installed and running:
+
+- A [Kubernetes cluster](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) running v1.11 or higher
+- [Kubectl CLI tool](https://kubernetes.io/docs/reference/kubectl/overview/) v1.10 or higher
+- [curl v7.65 or higher](https://curl.haxx.se/download.html)
+- Knative Eventing Component
+
+### Installing Knative Eventing 
+
+If you previously [created a Knative cluster](../install/_index.md), you might already have Knative Eventing installed and running. You can check to see if the Eventing component exists on your cluster by running:
+
+```sh
+kubectl get pods --namespace knative-eventing
+```
+
```

Comment:
Per the error i mention in another comment, we need to clarify that the "in-memory channel provisioner" is required and must be listed.